src/index.html

```diff
@@ -11,6 +11,42 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      data-qa="big"
+      method="post"
+      class="form"
+    >
+      <img 
+        src="images/Search.svg" 
+        alt="search"
+        class="icon-big"
+      >
+      <input
+        data-qa="keypress"
+        type="text"
+        name="bigSearch"
+        placeholder="Try &quot;Los Angeles&quot;"
+        class="input input-big"
+      >
+    </form>
+    <form
```

Comment:
Don't forget to add empty lines between multiline sibling blocks of HTML. You should check it everywhere 
```suggestion
    </form>
    
    <form
```